src/components/Layout.tsx

```diff
@@ -39,6 +41,23 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
     state.setConnectModal,
   ])
 
+  let concentratorsItem = (
+    <Menu.Item as={DropdownMenuItemLink} href={appLink("/yield/concentrators")}>
+      Concentrators
+    </Menu.Item>
+  )
+
+  if (DISABLE_CONCENTRATORS) {
+    concentratorsItem = (
+      <Menu.Item as={DropdownMenuItemLink} href={appLink("#")} disabled>
```

Comment:
You can just do `href="#"`. I think you should preventDefault when the user clicks this too though.

Reply:
Also, vertical alignment is just *slightly* off. If you put `flex items-center` on this it will be perfect.